contributing/style-guide.md

```diff
@@ -0,0 +1,12 @@
+# Pinot documentation style guide
+
+The guidelines below are specific to Apache Pinot's documentation. 
+ 
+ For general style questions or guidance on tpics not covered here, see the [Google style guide](https://developers.google.com/style) 
+
+## Star-tree index
+When describing the index type, use star-tree (adjective) or star tree (noun). StarTree refers to the company only. 
```

Comment:
curious is star tree noun a thing? if we haven't run into a specific use case, maybe best not to encourage/address this usage

Reply:
I have but only in one spot: https://docs.pinot.apache.org/basics/indexing/star-tree-index#definitions (first sentence here)

What do you think?

Reply:
Maybe 
The star-tree index stores data in a structure that consists of the following properties:


(and then under the graphic)
Star-tree index structure

Reply:
updated!